### DISCUSSION TITLE
[archived] "Steps Walk Through" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#9907] [General]
# argo-workflows/walk-through/steps/

https://argoproj.github.io/argo-workflows/walk-through/steps/

**Comment by xanalogica** [comment]
Please add the actual Argo CLI command and parameters used to generate this multi-step output report.

**Reply by jonwest** [reply]
It's just `argo get <workflow-name>`.

**Reply by agilgur5** [reply]
I added this to the page in #13157. 

Note that the Workflow `name` is not known beforehand as the Workflow uses `generateName`. Can use `argo get @latest` as per [below comment](https://github.com/argoproj/argo-workflows/discussions/9907#discussioncomment-6823248) if there haven't been any other Workflows submitted.

**Comment by skhurana333** [comment]
name is on UI or can use this command to get workflow list 
```sh
argo list
```

**Reply by skhurana333** [reply]
or for latest workflow info
```sh
argo get @latest
```
